<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InversePyramidDialog</class>
 <widget class="QDialog" name="InversePyramidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pyramidal reconstruction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="filterBox">
     <item>
      <property name="text">
       <string>triangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>trimodal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rectangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>qmf</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="nbStepLabel">
     <property name="text">
      <string>Number of steps in the pyramid : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="nbStepBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Step to reconstruct :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="stepBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>InversePyramidDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>InversePyramidDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
